bedrock/products/templates/products/vpn/resource-center/base.html

```diff
@@ -0,0 +1,163 @@
+        <a class='mzp-c-button mzp-t-product resource-center-cta'>Save $40 Today*</a>
+        <p class="resource-center-cta-desc">*When you subscribe to a 12 month plan</p>
+      </div>
+    </section>
+     <div class="mzp-l-content mzp-l-card-third">
+        {{ card (
+          title='Article title',
+          ga_title='Article title',
+          aspect_ratio='mzp-has-aspect-16-9',
+          desc='Wafer wafer wafer donut cake gingerbread bear claw ice cream. Fruitcake biscuit cotton candy cake pastry dessert.',
+          image_url="img/products/vpn/resource-center/resource-center-article-1.png",
+          link_url=url('products.vpn.resource-center.article')
+        )}}
+        {{ card (
+          title='Article title',
+          ga_title='Article title',
+          aspect_ratio='mzp-has-aspect-16-9',
+          desc='Wafer wafer wafer donut cake gingerbread bear claw ice cream. Fruitcake biscuit cotton candy cake pastry dessert.',
+          image_url="img/products/vpn/resource-center/resource-center-article-1.png",
+          link_url=url('products.vpn.resource-center.article')
+        )}}
+        {{ card (
+          title='Article title',
+          ga_title='Article title',
+          aspect_ratio='mzp-has-aspect-16-9',
+          desc='Wafer wafer wafer donut cake gingerbread bear claw ice cream. Fruitcake biscuit cotton candy cake pastry dessert.',
+          image_url="img/products/vpn/resource-center/resource-center-article-1.png",
+          link_url=url('products.vpn.resource-center.article')
+        )}}
+        {{ card (
+          title='Article title',
+          ga_title='Article title',
+          aspect_ratio='mzp-has-aspect-16-9',
+          desc='Wafer wafer wafer donut cake gingerbread bear claw ice cream. Fruitcake biscuit cotton candy cake pastry dessert.',
+          image_url="img/products/vpn/resource-center/resource-center-article-1.png",
+          link_url=url('products.vpn.resource-center.article')
+        )}}
+        {{ card (
+          title='Article title',
+          ga_title='Article title',
+          aspect_ratio='mzp-has-aspect-16-9',
+          desc='Wafer wafer wafer donut cake gingerbread bear claw ice cream. Fruitcake biscuit cotton candy cake pastry dessert.',
+          image_url="img/products/vpn/resource-center/resource-center-article-1.png",
+          link_url=url('products.vpn.resource-center.article')
+        )}}
+        {{ card (
+          title='Article title',
+          ga_title='Article title',
+          aspect_ratio='mzp-has-aspect-16-9',
+          desc='Wafer wafer wafer donut cake gingerbread bear claw ice cream. Fruitcake biscuit cotton candy cake pastry dessert.',
+          image_url="img/products/vpn/resource-center/resource-center-article-1.png",
+          link_url=url('products.vpn.resource-center.article')
+        )}}
+    </div>
+    {% include 'products/vpn/resource-center/includes/bottom-cta.html' %}
+
+    {% block js %}
```

Comment:
This block needs to move outside the `content` block (after the `{% endblock %}` below).